frontend/src/components/MealComponents/TotalMeals.component.js

```diff
@@ -8,69 +8,66 @@ import EditButton from "./edit.component";
 import DeleteButton from "./delete.component";
 import SelectMeal from "./select-meal.component";
 
-export default class TotalMeals extends Component {
-  constructor(props) {
-    super(props);
+const TotalMeals = () => {
+  const [meals, setMeals] = useState([]);
```

Comment:
Changed this from class to functional-based components, but somehow the meals page now throw an error - TypeError: meals.map is not a function. I've looked through and I confirmed that I'm mapping an array, also confirmed in my console. Please let me know what I'm doing wrong.